Documentation/CRDs/Cluster/external-cluster/external-cluster.md

```diff
@@ -275,3 +256,33 @@ you can export the settings from this cluster with the following steps.
 
 !!! important
     For other clusters to connect to storage in this cluster, Rook must be configured with a networking configuration that is accessible from other clusters. Most commonly this is done by enabling host networking in the CephCluster CR so the Ceph daemons will be addressable by their host IPs.
+
+## Upgrades
+
+1. If consumer cluster doesn't have restricted caps, this will upgrade all the default csi-users (non-restricted):
+
+    ```console
+    python3 create-external-cluster-resources.py --upgrade
+    ```
+
+2. If the consumer cluster has restricted caps:
+Restricted users created using `--restricted-auth-permission` flag need to pass mandatory flags: '`--rbd-data-pool-name`(if it is a rbd user), `--k8s-cluster-name` and `--run-as-user`' flags while upgrading, in case of cephfs users if you have passed `--cephfs-filesystem-name` flag while creating csi-users then while upgrading it will be mandatory too. In this example the user would be `client.csi-rbd-node-rookstorage-replicapool` (following the pattern `csi-user-clusterName-poolName`)
```

Comment:
It seems as though the text block is intended to be a paragraph under the bullet 'heading'. This will ensure it is indented properly on the web.
 
```suggestion
2. If the consumer cluster has restricted caps

    Restricted users created using `--restricted-auth-permission` flag need to pass mandatory flags: '`--rbd-data-pool-name`(if it is a rbd user), `--k8s-cluster-name` and `--run-as-user`' flags while upgrading, in case of cephfs users if you have passed `--cephfs-filesystem-name` flag while creating csi-users then while upgrading it will be mandatory too. In this example the user would be `client.csi-rbd-node-rookstorage-replicapool` (following the pattern `csi-user-clusterName-poolName`)
```